service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -718,6 +718,69 @@ public JobProperties exportDevices(String exportBlobContainerUri, Boolean exclud
         return future;
     }
 
+    /**
+     * Create a bulk export job.
+     *
+     * @param exportDevicesParameters A JobProperties object containing input parameters for export Devices job
+     *
+     * @return A JobProperties object for the newly created bulk export job
+     *
+     * @throws IllegalArgumentException This exception is thrown if the exportBlobContainerUri or excludeKeys parameters are null
+     * @throws IOException This exception is thrown if the IO operation failed
+     * @throws IotHubException This exception is thrown if the response verification failed
+     */
+    public JobProperties exportDevices(JobProperties exportDevicesParameters)
+            throws IllegalArgumentException, IOException, IotHubException, JsonSyntaxException
+    {
+        // CODES_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_15_062: [The function shall get the URL for the bulk export job creation]
+        URL url = iotHubConnectionString.getUrlCreateExportImportJob();
+
+        // CODES_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_15_063: [The function shall create a new SAS token for the bulk export job]
+        String sasTokenString = new IotHubServiceSasToken(this.iotHubConnectionString).toString();
+
+        // CODES_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_15_064: [The function shall create a new HttpRequest for the bulk export job creation ]
+        exportDevicesParameters.setType(JobProperties.JobType.EXPORT);
+        String jobPropertiesJson = exportDevicesParameters.toJobPropertiesParser().toJson();
```

Comment:
If this feature is off, you need to make sure the storageAuthenticationType is not serialized into the json, otherwise the service will reject the request.

Reply:
It is checked in toJobPropertiesParser. It won't be serialized into json